src/components/FormAlertWithSubmitButton.js

```diff
@@ -65,7 +65,6 @@ const FormAlertWithSubmitButton = props => (
         ) : (
             <Button
                 success
-                pressOnEnter
```

Comment:
Is it not safe to remove this prop completely from this PR? There are other components that need this. manually disable this via a prop for the components you have modified. 


When we have modified every component that uses this, we will remove it.